pkg/kubectl/cmd/util/openapi/errors.go

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
This should be done in a preliminary pull-request, along with a description. It's really hard to guess what you're trying to do and why you're doing this. (though I might have an idea :-)

Reply:
Also maybe move that code to a different package?